test/integration-test-main.sh

```diff
@@ -1309,6 +1309,196 @@ function test_rm_rf_dir {
    fi
 }
 
+function test_posix_acl {
+    describe "Testing posix acl function ..."
+
+    #------------------------------------------------------
+	# Directory
+    #------------------------------------------------------
+	local POSIX_ACL_TEST_DIR1="posix_acl_dir1"
+	local POSIX_ACL_TEST_DIR2="posix_acl_dir2"
+    mkdir "${POSIX_ACL_TEST_DIR1}"
```

Comment:
Could you fix the indentation in this file?  I think some tabs should be expanded to spaces.

Reply:
It's my fault, thus it is fixed.

Reply:
_additional comment_
There was still a missing fix. Fixed and rebased.